js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        let response = await this.publicGetTrades (this.extend ({
+            'market': symbol,
+            'limit': limit,
+        }, params));
+        let data = this.safeValue (response, 'data');
+        return this.parseTrades (data, symbol, since, limit);
+    }
+
+    parseTrade (trade, market = undefined) {
+        let side = undefined;
+        if (trade['side'] === 'bid') {
+            side = 'buy';
+        } else if (trade['side'] === 'ask') {
+            side = 'sell';
+        }
+        return {
+            'info': trade,
+            'timestamp': trade['created_on'],
+            'datetime': trade['created_at'],
+            'symbol': trade['market'],
+            'id': trade['id'],
+            'order': undefined,
+            'type': 'limit',
+            'takerOrMaker': undefined,
+            'side': side,
+            'price': this.safeFloat (trade, 'price'),
+            'amount': this.safeFloat (trade, 'volume'),
+            'cost': undefined,
+            'fee': undefined,
+        };
+    }
+
+    async fetchTime (params = {}) {
+        let response = await this.publicGetTimestamp ();
+        return response['data'];
+    }
+
+    async fetchTradingFees (params = {}) {
+        let response = await this.publicGetFeesTrading ();
+        return response['data'];
+    }
+
+    async fetchKey (params = {}) {
+        // private method input value {'user_jwt': encrypted_jwt_token}
+        // jwt_token should be RS256 encoded
+        let response = await this.privateGetKey (this.extend (params));
+        return response['data'];
+    }
+
+    async fetchBalance (params = {}) {
+        let response = await this.privateGetMembersMe (this.extend (params));
+        const balances = this.safeValue (response['data'], 'accounts');
+        let result = { 'info': balances };
+        for (let i = 0; i < balances.length; i++) {
+            const balance = balances[i];
+            let currency = balance['currency'];
+            let account = this.account ();
+            let free = this.safeFloat (balance, 'balance');
+            let used = this.safeFloat (balance, 'locked');
+            let total = free + used;
+            account['free'] = free;
+            account['used'] = used;
+            account['total'] = total;
+            result[currency] = account;
+        }
+        return this.parseBalance (result);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        let response = await this.privatePostOrders (this.extend (params));
+        const data = response['data'];
+        let result = {
+            'info': data,
+            'id': data['id'],
+            'symbol': data['market'],
+            'type': data['ord_type'],
+            'side': data['side'],
+            'status': data['state'],
+        };
+        return result;
```

Comment:
We need to pass the response to `parseOrder` to avoid duplicating the parsing code here, so we need to remove the result structure from line 227 and add `return this.parseOrder (data)`.